.Lib9c.Tests/Util/CraftUtil.cs

```diff
@@ -0,0 +1,75 @@
+namespace Lib9c.Tests.Util
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using Libplanet;
+    using Libplanet.Action;
+    using Nekoyume;
+    using Nekoyume.Action;
+    using Nekoyume.Model;
+    using Nekoyume.Model.Item;
+    using Nekoyume.Model.State;
+    using Nekoyume.TableData;
+    using static Lib9c.SerializeKeys;
+
+    public static class CraftUtil
+    {
+        public static IAccountStateDelta PrepareCombinationSlot(
+            IAccountStateDelta state,
+            Address avatarAddress,
+            int slotIndex
+        )
+        {
+            var slotAddress = avatarAddress.Derive(string.Format(
+                CultureInfo.InvariantCulture,
+                CombinationSlotState.DeriveFormat,
+                slotIndex));
+            var slotState = new CombinationSlotState(
+                slotAddress,
+                // CombinationEquipment: 3
+                // CombinationConsumable: 6
+                // ItemEnhancement: 9
+                GameConfig.RequireClearedStageLevel.ItemEnhancementAction
```

Comment:
Originally this code is written by me. I want review for this code.
Is it better to give action type from function caller?

Reply:
Yes that's right! We can apply it next time.🙂